src/axes/axis.ts

```diff
@@ -273,6 +273,9 @@ export class Axis<D> extends Component {
     if (annotationTierCount == null) {
       return this._annotationTierCount;
     }
+    if (annotationTierCount < 0) {
+      throw new Error(`annotationTierCount ${annotationTierCount} is not valid, should be positive or 0`);
```

Comment:
I personally think this can be made more concise and it would be better with one of the following ways:
* "annotationTierCount must be positive or 0"
* "annotationTierCount cannot be negative"
* "annotationTierCount ${annotationTierCount}..." with any of the above.